app/src/main/java/grinnell/appdev/edu/lyles/fragments/FavoritesFragment.java

```diff
@@ -1,22 +1,113 @@
 package grinnell.appdev.edu.lyles.fragments;
 
 import android.os.Bundle;
-import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ListView;
 
+import org.json.JSONArray;
+
+import java.util.ArrayList;
+import java.util.concurrent.ExecutionException;
+
+import grinnell.appdev.edu.lyles.AsyncRetrieval;
+import grinnell.appdev.edu.lyles.ItemAdapter;
+import grinnell.appdev.edu.lyles.JSONConstants;
+import grinnell.appdev.edu.lyles.MenuItem;
 import grinnell.appdev.edu.lyles.R;
+import grinnell.appdev.edu.lyles.preferences.FavoritesManager;
 
 /**
  * Created by Mattori on 5/9/16.
  */
 public class FavoritesFragment extends Fragment {
 
+    private ArrayList<String> mAllURLs;
+    private ArrayList<String> mAllArrayTitles;
+
+    private ArrayList<MenuItem> mAllMenuItems; // Contains all menu items in all categories
+    private ItemAdapter mItemAdapter;
+    private RecyclerView mRecyclerView;
+
+    private FavoritesManager mFavoritesManager;
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
```

Comment:
Now our `onCreateView(...)` is really long. Why don't we make it modular by taking out some of the things happening here and make them methods. Our goal is to keep our methods short, concise and straight to the point (i.e it should do **one thing** and one thing only)